apps/judicial-system/web/src/shared-components/PageLayout/utils/index.ts

```diff
@@ -0,0 +1,108 @@
+import {
+  Case,
+  CaseDecision,
+  CaseState,
+  CaseType,
+  isInvestigationCase,
+  isRestrictionCase,
+} from '@island.is/judicial-system/types'
+import {
+  getCourtSections,
+  getCustodyAndTravelBanProsecutorSection,
+  getExtenstionSections,
+  getInvestigationCaseCourtSections,
+  getInvestigationCaseProsecutorSection,
+} from '../Sections'
+
+interface TranslationStrings {
+  dismissedTitle: string
+}
+
+export const caseResult = (
+  translationStrings: TranslationStrings,
+  workingCase?: Case,
+) => {
+  if (!workingCase) {
+    return ''
+  }
+
+  const isRejected =
+    workingCase?.state === CaseState.REJECTED ||
+    workingCase?.parentCase?.state === CaseState.REJECTED
```

Comment:
I don't fully understand the logic here, but think it is strange to check if a parent case is rejected. Only accepted cases can be extended, so it follows that the parent case is always accepted.

Reply:
You are absolutely right, thanks 🙏